conference/react-js/src/actions/getTalksLangLevels.js

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
+import { langLevels } from '../constants/backend-url';
```

Comment:
Should we have this 5 same files? Why not create one file 'Talk' with  all actions?